repo/mk/core.misc.channels.deps.mk

```diff
@@ -0,0 +1,27 @@
+# ------------------------------------------------------------------------------
+
+SF_DEPS_TARGETS += \
+	deps-channels \
+
+# ------------------------------------------------------------------------------
+
+.PHONY: deps-channels
+deps-channels:
```

Comment:
help string here maybe?

Reply:
I dont think we usually annotate the targets that arent meant to be used directly. This one gets added to the targets than run under `make deps` so it shouldnt need to be run manually